fed_test_utils/fed.go

```diff
@@ -153,51 +154,89 @@ func NewFedTest(t *testing.T, originConfig string) (ft *FedTest) {
 	directorStartTime := time.Now().Add(-6 * time.Minute)
 	director.SetStartupTime(directorStartTime)
 
-	config.InitConfig()
 	err = config.InitServer(ctx, modules)
 	require.NoError(t, err)
 
 	// Read in any config we may have set
+	var originConf interface{}
```

Comment:
It gets a bit confusing to have both `originConf` and `originConfig` -- can you try another variable name? Also, I think we can use `any` instead of `interface{}` here, which is a bit easier to read in my opinion.